jsonschema-rs-iter/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "jsonschema-rs-iter"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+jsonschema = "0.13"
+structopt = {version = ">= 0.3"}
```

Comment:
We should use `clap`, `structopt` is now in maintenance mode.

Reply:
ok done in next version